exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -103,10 +103,11 @@ hugo new posts/first_post.md
 
 Feel free to edit the post file by adding some sample content and replacing the title value in the beginning of the file.
 
-For posts you need to add `type: "post"` in the markdown metadata. We currently support 2 types of content:
+For posts you need to add `type: "post"` in the markdown metadata. We currently support 3 types of content:
 
 1. Post (`type: "post"`): A normal blog-post with tags, date, & content.
 2. Page (`type: "page"`): A standalone content page that will just render the markdown you wrote. You can use it to write custom pages which should not be a part of posts. Like showing your projects portfolio. You can read in detail about this on the [Theme Documentation - Advanced](/posts/theme-documentation-advanced/#content-types) page.
+3. About (`type: "about"`): This is for displaying more content on the homepage rather than just a single line. It's a drop-in replacement for the `description` field from the configuration. See [Theme Documentation - About description text](/posts/theme-documentation-advanced/#about-description-text).
```

Comment:
Do we need this 3rd type ? Since its only for the homepage, can we just infer it via filename and its existense ?